Fix typo in xyframe.tex, patch by Andrey Paramonov, see Debian #561784
---
 texmf-dist/tex/generic/xypic/xyframe.tex |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: texlive-base-2011.20120216/texmf-dist/tex/generic/xypic/xyframe.tex
===================================================================
--- texlive-base-2011.20120216.orig/texmf-dist/tex/generic/xypic/xyframe.tex	2011-08-03 03:42:25.000000000 +0900
+++ texlive-base-2011.20120216/texmf-dist/tex/generic/xypic/xyframe.tex	2012-02-17 00:01:21.889521262 +0900
@@ -517,7 +517,7 @@
 \xydef@\UseCurveFrames@{%
  \let\circled@x@@=\xycircleframe@x@
  \let\ellipsed@x@@=\xyellipseframe@x@ }
-\def\UseCurveFrames{\xywithoption{curve}{\UseCurveframes@}}%
+\def\UseCurveFrames{\xywithoption{curve}{\UseCurveFrames@}}%
 \xydef@\UseFontFrames@{%
  \let\circled@x@@=\circled@x@
  \let\ellipsed@x@@=\ellipsed@x@ }
